build/scripts/docker/Dockerfile

```diff
@@ -30,6 +30,14 @@ RUN apt-get install -y  libunwind8 \
             libcurl4-openssl-dev && \
     apt-get clean
 
+# Install Mono
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF && \
+    (echo "deb http://download.mono-project.com/repo/ubuntu stable-xenial main" | \
+    tee /etc/apt/sources.list.d/mono-official.list) && \
+    apt-get update && \
+    apt-get install -y mono-devel=5.8.0.108-0xamarin1+ubuntu1604b1 && \
```

Comment:
How long do builds like this stay around? I.E. will we be able to run this build 1 year, 2 years, etc ... from now

Reply:
That's a reasonable question for the Mono people. I can't find any statement of how long they keep builds, but I do see builds many years old on their server. For instance, http://download.mono-project.com/repo/ubuntu/dists/stable-wheezy/snapshots/ lists builds from 2014. This is the stable, not nightly repo, so I expect this package to live for quite a while -- probably as long as the lifetime of the Ubuntu 16.04, which is 5 years since this is an LTS distribution.